lib/pure/net.nim

```diff
@@ -1648,6 +1660,31 @@ proc recvFrom*(socket: Socket, data: var string, length: int,
   else:
     raise newException(ValueError, "Unknown socket address family")
 
+proc recvFrom*(socket: Socket;
+               data: var string, length: int;
+               address: var IpAddress, port: var Port,
+               flags = 0'i32): int {.tags: [ReadIOEffect].} =
+  ## Receives data from `socket`. This is similar to the previous
+  ## proc but takes an IpAddress and also returns the bytes written.
```

Comment:
"previous proc" - can you link to the proc instead? :)

Reply:
Ah! I didn't realize that was possible in rst. I'm going to try the combined form mentioned above.